src/core/http.ts

```diff
@@ -10,22 +10,35 @@ import {
 } from './handlers/HttpHandler'
 import type { Path, PathParams } from './utils/matching/matchRequestUrl'
 
+export type HttpRequestHandler = <
```

Comment:
## Todo

We don't have any tests for this type right now. Need to add it to make sure it compiles without issues.

Reply:
Added the tests.